packages/esm-patient-list-app/src/Modal.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import style from './style.scss';
+
+const Modal: React.FC<{ close: () => void }> = ({ children, close }) => {
```

Comment:
I think this one can be removed (was replaced by the general one from the app shell)